src/pages/DevOps.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Fragment } from 'react';
+import { Button, Card, CardBody, Col, Row } from 'reactstrap';
+import ReactPlayer from 'react-player';
+import { useRouteData } from 'react-static';
+import { Link } from '@reach/router';
+import student1 from '../assets/images/photo1.jpg';
+import logophoto from '../assets/images/logodevops.jpg';
+
+const Dev = () => {
```

Comment:
```suggestion
const DevOps = () => {
```
Component names should match file names